### PR TITLE
Add VMBoundPort workbooks

### PR DESCRIPTION
**Pull request by andrewki-msft**
Add VMBoundPort workbooks for VM Insights AtScale workbooks gallery

Active Ports
image

Open Ports
image

### CODE REVIEW

File: Workbooks/Virtual Machines - Network Dependencies/Open Ports/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name": "Open Ports",
+    "author": "Microsoft",
+    "galleries": [
+        { "type": "workbook", "resourceType": "microsoft.operationalinsights/workspaces", "order": 200 }
```
Comment (acearun) [inline Workbooks/Virtual Machines - Network Dependencies/Open Ports/settings.json]: This is the same order as the other template. Change it up or down based on priority.

File: Workbooks/Virtual Machines - Network Dependencies/Active Ports/settings.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Active Ports",
```
Comment (acearun) [inline Workbooks/Virtual Machines - Network Dependencies/Active Ports/settings.json]: I can't tell the difference between Active Ports and Open Ports. Maybe a better name would help.

**Comment by acearun** [comment]
In the Open Ports report, surely the computers don't have thousands of ports open? Do you have the wrong aggregation in the query?

**Comment by acearun** [comment]
In the Open Ports report, you have the option of showing a hierarchy of Process > Open Ports with some metric (like bytes sent). The flat list is a bit bland.

File: Workbooks/Virtual Machines - Network Dependencies/Open Ports/settings.json (hunk 1 above)
Comment (andrewki-msft) [reply Workbooks/Virtual Machines - Network Dependencies/Open Ports/settings.json]: Updated order of workbooks
image

File: Workbooks/Virtual Machines - Network Dependencies/Active Ports/settings.json (hunk 2 above)
Comment (andrewki-msft) [reply Workbooks/Virtual Machines - Network Dependencies/Active Ports/settings.json]: Those were the names that our PM created them with, unfortunately changing the titles would incur changes elsewhere (e.g. blog, etc.) so will leave this be for now.

**Comment by andrewki-msft** [comment]
> In the Open Ports report, surely the computers don't have thousands of ports open? Do you have the wrong aggregation in the query?

Yes, you're right, will fix this

**Pull request merged**